lib/server.js

```diff
@@ -2013,6 +2027,20 @@ WalletService.prototype.createTx = function(opts, cb) {
         self._selectTxInputs(txp, opts.utxosToExclude, next);
       },
       function(next) {
+        self._getNetworkParams(wallet.network, function(err, params) {
+          if (err) return next(err);
+
+          var minRelayFee = params.minRelayFee || Defaults.MIN_RELAY_FEE_SAT_PER_BYTE;
+          var dustThreshold = Math.round(3 * minRelayFee * txp.getEstimatedSize() * (1 + Defaults.DUST_MARGIN));
```

Comment:
Shouldn't this be:
```javascript
var dustThreshold = Math.round(3 * minRelayFee * 182 * (1 + Defaults.DUST_MARGIN));
```
dust threshold is not a function of any transaction, it is a function of the minRelayTx under the assumption of the smallest possible contribution an output could make to the byte size... (1 34 byte output and 148 byte input... so 182 bytes)